rb/lib/selenium/webdriver/common/log_entry.rb

```diff
@@ -22,22 +22,22 @@ module WebDriver
     class LogEntry
       attr_reader :level, :timestamp, :message
 
-      def initialize(level, timestamp, message)
-        @level = level
+      def initialize(timestamp, level, message)
```

Comment:
Is there any value to change the order here? Even though nobody should be initializing this class directly, I don't believe it's worth breaking backwards compatibility without a good reason.

Reply:
I'll undo this change, but it was only used in the entire codebase in one location, which was api private.

Reply:
Sorry yes. The other logger formats in a different way, and the timestamp is always first in most loggers. Here it wasn't.